tools/public_api.h

```diff
@@ -51,6 +52,7 @@ int HPyList_Append(HPyContext ctx, HPy h_list, HPy h_item);
 /* dictobject.h */
 HPy HPyDict_New(HPyContext ctx);
 int HPyDict_SetItem(HPyContext ctx, HPy h_dict, HPy h_key, HPy h_val);
+HPy HPyDict_GetItem(HPyContext ctx, HPy h_dict, HPy h_key);
```

Comment:
this is probably "wrong": we don't want API which return borrowed references. Please add a comment so we remember to fix

Reply:
Comment added!